examples/react-full/renderer/pageContext.d.ts

```diff
@@ -0,0 +1,24 @@
+import 'vite-plugin-ssr';
+
+declare module "vite-plugin-ssr" {
+  namespace VitePluginSsr {
+    interface PageContextInit {
+      pageExports: {
+        documentProps?: {
+          title: string
+        }
+      }
+    }
+
+    // Is it possible to also support `pageContext` on the browser-side by doing something like the following?
```

Comment:
Thoughts? So that we can also provide `PageContextInit` to [`getPage()`](https://vite-plugin-ssr.com/getPage) and [`useClientRouter()`](https://vite-plugin-ssr.com/useClientRouter).

Reply:
It would be great, I'll need to take a look at it

Reply:
Let me check that once this PR is merged or stable enough to be merged

Reply:
Ok